<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTemplate</class>
 <widget class="QWidget" name="AddTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(227, 227, 227);
border:1px solid rgb(180,180,180);
border-radius:5px;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(180,180,180);
border-top-right-radius:5px;
border-top-left-radius:5px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;
border:none</string>
    </property>
    <property name="text">
     <string>添加模板</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnClose">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:5px;
border-top-left-radius:0px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>34</y>
     <width>660</width>
     <height>366</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:0px;
border-top-left-radius:0px;
border-bottom-right-radius:5px;
border-bottom-left-radius:5px;</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
font: 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>选择照片对应目标人</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
font: 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>照片名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
font: 11pt &quot;等线&quot;;</string>
    </property>
    <property name="text">
     <string>照片描述</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_labelImage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:2px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnCancel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
border: none;
border-radius: 2px;
background-color:rgb(180,180,180)</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnAddImage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
border: none;
border-radius: 2px;
background-color:rgb(180,180,180)</string>
    </property>
    <property name="text">
     <string>添加照片</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_editPerson">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:2px;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_editImageName">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:2px;
background-color: rgb(255, 255, 255);
font: 18px &quot;等线&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_labelImageDes">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:2px;
background-color: rgb(255, 255, 255);
font: 18px &quot;等线&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnSave">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
border: none;
border-radius: 2px;
background-color:rgb(180,180,180)</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
